src/plugins/analysis/binwalk/install.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+import urllib.request
+
+from helperFunctions.install import run_cmd_with_logging
+
+from ...installer import AbstractPluginInstaller
+
+# The base directory of the plugin
+base_path = pathlib.Path(__file__).resolve().parent
```

Comment:
It is used by `install/backend.py` to figure out the path of the plugin. While this could be replaced, we need the base path for making the installers be able to run standalone.